exporter/signalfxexporter/translation/dpfilters/metricfilter.go

```diff
@@ -30,7 +30,7 @@ type MetricFilter struct {
 func (mf *MetricFilter) normalize() (map[string][]string, error) {
 	if mf.MetricName != "" {
 		if mf.MetricNames == nil {
-			mf.MetricNames = make([]string, 1)
+			mf.MetricNames = make([]string, 0, 1)
```

Comment:
🤔 why does it need to initialize an empty slice here anyway?

Reply:
Good point! It does not. Fixed.